data/virtualization/add_before.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
suggest to add maintainer info for this script

Reply:
Good idea!